ten-simple-rules-dockerfiles.Rmd

```diff
@@ -97,12 +97,12 @@ For this article, we focus on workflows that typically run on single machine, e.
 Readers might scope the data requirement to less than a terabyte and the compute requirement to a machine with 16 cores running over the weekend.
 
 Although it is outside the scope of this article, we point readers to `docker-compose` [@docker-compose_2019] in the case where one might need container orchestration for multiple applications, e.g., web servers, databases, and worker containers. 
-A `docker-compose.yml` configuration file allows for defining volumes, environment variables, and exposed ports and helps users stick to  _one purpose per container_, which often means one process running in the container, and to combine existing stable building blocks instead of bespoke massive containers for specific purposes.
+A `docker-compose.yml` configuration file allows for defining volumes, environment variables, and exposed ports and helps users stick to _one purpose per container_, which often means one process running in the container, and to combine existing stable building blocks instead of bespoke massive containers for specific purposes.
 
 Because _"the number of unique research environments approximates the number of researchers"_ [@nust_opening_2017], sticking to conventions helps every researcher to understand, modify, and eventually write container recipes suitable for their needs.
 Even if they are not sure how the underlying technology actually works, researchers should leverage containerisation following good practices.
 The practices that are to be discussed in this article are strongly related to software engineering in general and research software engineering (RSEng) in particular, which is concerned with quality, training, and recognition of software in science [@cohen_four_2020].
-We encourage you to reach out to your local or national RSE (CB comment: Should this abbreviation be RSEng?) community if you have questions on software development in research that go beyond the rules of this work.
+We encourage you to reach out to your local or national RSE (CRB comment: Should this abbreviation be "RSEng"?) community if you have questions on software development in research that go beyond the rules of this work.
```

Comment:
Yes, because #RSE hashtag on Twitter corresponds to "Relationship and Sex Education." heh

Reply:
In the sentence before we introduce `RSEng` as an abbreviation, so I'd say: yes. Let's just use one of the two.